ext/crypto/00_crypto.js

```diff
@@ -1071,9 +1458,150 @@
 
               return key;
             }
+            case "jwk": {
+              // 2. https://w3c.github.io/webcrypto/#rss-oaep-operations
+              let hash;
+
+              const jwk = keyData;
+              // 2-3.
+              if (
+                ArrayPrototypeFind(
+                  keyUsages,
+                  (u) =>
+                    !ArrayPrototypeIncludes(
+                      jwk.d ? ["decrypt"] : ["encrypt"],
+                      u,
+                    ),
```

Comment:
`unwrapKey` and `wrapKey` are valid too

Reply:
Ok. I understood that the general idea with CryptoAPI was not to accept anything that was not yet implemented, such as wrapKey and unwrapKey. That's why I excluded them. I see however that pkcs8 import has them.